include/snowflake/sb_CRTFunctionSafe.h

```diff
@@ -21,186 +17,257 @@
 extern "C" {
 #endif
 
-#ifndef STDCALL
-    // match snowflake define in platform.h
-    #ifdef _WIN32
-        #define STDCALL __stdcall
-    #else
-        #define STDCALL
-    #endif
-#endif
-
-extern char* STDCALL sf_getenv(const char *name);
-extern char* STDCALL sf_strerror(int);
+#ifndef STDCALL
+    // match snowflake define in platform.h
+    #ifdef _WIN32
+        #define STDCALL __stdcall
+    #else
+        #define STDCALL
+    #endif
+#endif
+
+extern char* STDCALL sf_getenv(const char *name, char *outbuf, size_t bufsize);
+extern char* STDCALL sf_strerror(int errnum, char* outbuf, size_t bufsize);
+
+    /// @brief Copy bytes between buffers.
+    ///
+    /// @param out_dest         Destination buffer. (NOT OWN)
+    /// @param in_destSize      Size of the destination buffer.
+    /// @param in_src           Buffer to copy from. (NOT OWN)
+    /// @param in_sizeToCopy    Number of bytes to copy.
+    ///
+    /// @return A pointer to destination; NULL if an error occurred. (NOT OWN)
+    static inline void* sb_memcpy(
```

Comment:
should it be `sb_` or `sf_`? and why this method declaration has additional indentation?